src/app/routes/idx/index.js

```diff
@@ -0,0 +1,10 @@
+import { IdxPage } from '#pages';
+import { IdxPagePath } from '../utils/regex';
+
+export default {
+  path: IdxPagePath,
+  exact: true,
+  component: IdxPage,
+  getInitialData: () => Promise.resolve({ status: 200, pageData: {} }),
+  pageType: 'idx',
```

Comment:
I think this should be in capitals to match what is in the RequestContext props

```suggestion
  pageType: 'IDX',
```

Reply:
I am now in a small rabbit hole looking at this, will report back when I've climbed out

Reply:
@thekp is correct though!

Reply:
What confused me isn't relevant to this conversation, I'll write it up somewhere. tl;dr I think we should have these defined as constants and use them in both places if they have to match, but we can do that in another PR.

Reply:
We already have a file with some constants in, but it's not being used for `pageType` in the routes files, or for the prop types in request context. I'll create an issue to change that :)